docs/mine-hnt/validators/wallet.mdx

```diff
@@ -30,6 +30,14 @@ At any time use ```-h``` or ```--help``` to get more help for a command.
 
 ```./helium-wallet create basic --network testnet``` 
 
+#### Create a Wallet in a non-interactive / DevOps way
+
+```
+export HELIUM_WALLET_PASSWORD="My Password"
+./helium-wallet create basic --network testnet
```

Comment:
```suggestion
HELIUM_WALLET_PASSWORD="My Password" ./helium-wallet create basic --network testnet
```

Reply:
I would recommend this one  liner instead

Reply:
Precisely like this. The importance of not exporting HELIUM_WALLET_PASSWORD lies in the fact once the process has finished, the password will remain in memory. However, in the change proposed here (without the `export` command, the variable will be only availble to the helium-wallet process and nothing else.

Reply:
I think we're in agreement. Removing the `export` line and injecting it during the executable invocation is what I've proposed  here. I'm waiting for you to accept the suggestions since it's your PR.